os/syscall/syscall_stublookup.c

```diff
@@ -58,6 +58,7 @@
 
 #include <sys/types.h>
 #include <syscall.h>
+#include <syscall_not_support.h>
```

Comment:
This is a local header. We need to use quotes in the include statement. 
`#include "syscall_not_support.h"`